src/jammed.js

```diff
@@ -401,6 +385,141 @@ Method.prototype.draw = function(x, y) {
   }
   text += ")";
   apo.ctx.fillText(text, x+10+len.width, y);
+};console.log('loaded samplejs');
+
+function draw() {
+  console.log('draw called');
+  var canvas = document.getElementById('samplecanvas');
+  if (!canvas.getContext) {
+    console.log("Canvas not supported");
+    return;
+  }
+  var ctx = canvas.getContext('2d');
+  lines(ctx);
+  text(ctx);
+//
+//  ctx.fillRect(25,25,100,100);
```

Comment:
I know these comments aren't from the pull request, but they can be removed from src/test.js.